Task 4/Task4/Task4/CastomGit.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Actions;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Task4
+{
+    class CastomGit
+    {
+        private string path;
+
+        private List<(DateTime, FileAction)> Log = new List <(DateTime, FileAction)>();
+
+        public Dictionary<String, String> FilePastPath = new Dictionary<string, string>();
+
+        private UserCommunication communicator = new UserCommunication();
+
+        private DateTime LogTime;
+
+        public CastomGit(string path)
+        {
+            this.path = path;
+        }
+
+        
+        public void StartProgram()
+        {
+            string[] files = Directory.GetFiles(path, "*.txt");
+
+            DateTime currentTime = DateTime.Now;
+
+            string initialCommitPath = path + "\\" + currentTime.ToString("dd.MM.yyyy.HH.mm.ss");
+
+            Directory.CreateDirectory(initialCommitPath);
```

Comment:
So it creates many subfolders in the folder you monitor? Also you don't make them hidden